modules/test/playwright/tests/frontend-data-set-views-web/fields.spec.ts

```diff
@@ -75,6 +75,28 @@ test.describe('Add fields to a view and show them in a fragment', () => {
 			await expect(page.getByText('creator.id').first()).toBeVisible();
 			await expect(page.getByText('creator.name').first()).toBeVisible();
 		});
+
+		await test.step('Open modal to edit a field', async () => {
+			const dropdownMenuButton = page
+				.locator('.actions-cell button')
+				.first();
```

Comment:
How do you know what field is first? Isn't `dateCreated` first, that should have sorting option enabled? I am more and more certain that we should avoid using `first()` whenever possible.